Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
```

Comment:
No, the title wouldn't be set. I'm going to attach 2 pictures to show the difference, and you tell me which you would prefer:

![detailviewcontroller w title](https://cloud.githubusercontent.com/assets/7132954/14589466/6ada5e18-04a8-11e6-9a69-a5e3348c7f78.png)
![detailviewcontroller w o title](https://cloud.githubusercontent.com/assets/7132954/14589467/7051ddbc-04a8-11e6-924d-f39e1ebe0913.png)

The only thing that changes is that on the top of the screen we have the title of the event show up twice, as opposed to having the title once and the view controller's title as "Details"